scRNA-seq-advanced/02-dataset_integration.Rmd

```diff
@@ -739,8 +739,8 @@ Like `fastMNN`, `harmony` performs integration on a merged PCA matrix.
 However, unlike `fastMNN`, `harmony` does not "back-calculate" corrected expression from the corrected PCA matrix and it only returns the corrected PCA matrix itself.
 For input, `harmony` needs a couple pieces of information:
 
-- First, `harmony` takes batch-weighted PCA matrix directly to perform integration.
-Since we already calculated a batch-weighted PCA matrix (our `merged_PCA` reduced dimension), we'll provide this information directly.
+- First, `harmony` takes a batch-weighted PCA matrix to perform integration.
+We already calculated a batch-weighted PCA matrix (our `merged_PCA` reduced dimension), we'll provide this variable.
```

Comment:
You don't provide a variable, and here especially not, as you are using a function call to get the reduced dimension matrix as input.
```suggestion
We already calculated a batch-weighted PCA matrix (our `merged_PCA` reduced dimension), we'll provide this as the the input.
```